src/screens/DomainScreen/index.js

```diff
@@ -125,10 +126,12 @@ const DomainScreen = () => {
       }
 
       await getDomainFeed(postOffset);
+    } else {
+      await getDomainFeed(postOffset);
+    }
 
-      if (withLoading) {
-        setLoading(false);
-      }
+    if (withLoading) {
+      setLoading(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces redundancy by calling `getDomainFeed(postOffset);` regardless of whether the condition `domainName !== domainStore.selectedLastDomain` is true or false. The old code only called this function once after the conditional block. Consider removing the redundant call.
```diff
-      await getDomainFeed(postOffset);
-    } else {
-      await getDomainFeed(postOffset);
-    }
-
-    if (withLoading) {
+    }
+
+    await getDomainFeed(postOffset);
+
+    if(withLoading) {
       setLoading(false);
     }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->